compiler/loco/CMakeLists.txt

```diff
@@ -14,6 +14,8 @@ target_link_libraries(loco PRIVATE nncc_common)
 target_link_libraries(loco PUBLIC nncc_coverage)
 # Q. HOW TO MAKE DEV PACKAGE(?)
 install(TARGETS loco DESTINATION lib)
+install(DIRECTORY include/ DESTINATION include
+        FILES_MATCHING PATTERN "*.h")
```

Comment:
What is this for?

Reply:
Are you going to introduce developer package?

Reply:
Oh, CMake 3.5 support this pattern ;-) 
- [Reference document](https://cmake.org/cmake/help/v3.5/command/install.html#installing-files).

Reply:
@parjong 
> Are you going to introduce developer package?

Right. This is for `one-compiler-dev` debian package:)